bcc/module.go

```diff
@@ -412,6 +412,24 @@ func (bpf *Module) AttachMatchingUretprobes(name, match string, fd, pid int) err
 	return nil
 }
 
+func (bpf *Module) GetSyscallFnName(name string) string {
```

Comment:
Since the module is not needed, can you change `GetSyscallFnName` and `GetSyscallPrefix` like follows?

```
diff --git a/bcc/module.go b/bcc/module.go
index b312cb6..a8ccb19 100644
--- a/bcc/module.go
+++ b/bcc/module.go
@@ -412,11 +412,11 @@ func (bpf *Module) AttachMatchingUretprobes(name, match string, fd, pid int) err
        return nil
 }
 
-func (bpf *Module) GetSyscallFnName(name string) string {
-       return bpf.GetSyscallPrefix() + name
+func GetSyscallFnName(name string) string {
+       return GetSyscallPrefix() + name
 }
 
-func (bpf *Module) GetSyscallPrefix() string {
+func GetSyscallPrefix() string {
        _, err := bccResolveName("", "sys_bpf", -1)
        if err == nil {
                return "sys_"
```

Reply:
good call, I made the change.